styles/global.js

```diff
@@ -23,13 +23,53 @@ export default css.global`
   html {
     background: white;
     box-sizing: border-box;
-    padding: 2% 5%;
+    //padding: 2% 5%;
   }
 
   main {
     display: flex;
     flex-direction: column;
     align-items: center;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+  }
+
+  .navbar {
+    width: 100%;
+    margin: 0 0 3% 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    background-color: var(--pink)
+  }
+
+  .brand-logo {
+    display: flex; 
+    flex-direction: column; 
+    align-items: flex-start;
+    justify-content: center;
+    margin-left: 2rem;
+    color: #fff;
+  }
+
+  .navbar ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+
+  .navbar ul li a {
+    margin-right: 2rem;
+    font-weight: 500;
+    color: #fff;
+  }
+
+  .navbar ul li a:hover {
+    text-decoration: underline;    
```

Comment:
What do you think about making a `Header` component and making a CSS file for that component? I'd like to move away from using this global styles file. It won't scale well as more people start contributing. You could put the CSS in a file called `Header.module.css`. Here's the Next.js docs on how to define component-level CSS https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css.